libs/sdk-ui-ext/src/internal/constants/uiConfig.ts

```diff
@@ -4,22 +4,22 @@ import { IUiConfig } from "../interfaces/Visualization.js";
 import { METRIC, FACT, ATTRIBUTE, DATE, GEO_ATTRIBUTE } from "./bucket.js";
 import { BucketNames, OverTimeComparisonTypes, VisualizationTypes } from "@gooddata/sdk-ui";
 
-export const MAX_METRICS_COUNT = 40;
-export const DEFAULT_PIE_METRICS_COUNT = 1;
-export const DEFAULT_PIE_ONLY_MAX_METRICS_COUNT = 20;
-export const DEFAULT_TREEMAP_MEASURES_COUNT = 1;
-export const MAX_FILTERS_COUNT = 20;
-export const MAX_CATEGORIES_COUNT = 1;
-export const MAX_TABLE_CATEGORIES_COUNT = 20;
-export const INCREASE_MAX_TABLE_MEASURE_ITEMS_LIMIT = 100;
-export const INCREASE_MAX_TABLE_ATTRIBUTES_ITEMS_LIMIT = 50;
-export const MAX_STACKS_COUNT = 1;
-export const MAX_VIEW_COUNT = 2;
-export const DEFAULT_HEADLINE_METRICS_COUNT = 1;
-export const DEFAULT_XIRR_METRICS_COUNT = 1;
-export const DEFAULT_XIRR_ATTRIBUTES_COUNT = 1;
-const DEFAULT_GEO_ATTRIBUTES_COUNT = 1;
-const DEFAULT_PUSHPIN_METRICS_COUNT = 1;
+export const MAX_METRICS_COUNT = 1_000;
+export const DEFAULT_PIE_METRICS_COUNT = 1_000;
```

Comment:
change all const devinition to validn number not shure about ecma script 2021
Pls change it export const DEFAULT_PIE_METRICS_COUNT = 1000;